velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -122,6 +122,8 @@ void registerArrayFunctions(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_array_sort, prefix + "array_sort");
   VELOX_REGISTER_VECTOR_FUNCTION(
       udf_array_sort_desc, prefix + "array_sort_desc");
+  VELOX_REGISTER_VECTOR_FUNCTION(
```

Comment:
Perhaps, move this registration into AggregationFuzzerTest. This function should not be available in production setup.